azure/packages/test/end-to-end-tests/package.json

```diff
@@ -12,8 +12,8 @@
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
 	"scripts": {
-		"build": "concurrently npm:build:compile npm:lint",
-		"build:compile": "npm run build:test",
+		"build": "fluid-build . --task build",
+		"build:compile": "fluid-build . --task compile",
```

Comment:
Concern: 

What happens if by mistake we remove the `build:test` script in this package? Not the best example because that would probably cause an obvious error in CI, but assuming for argument's sake that it didn't (say it was a formatting script)... would we just not know that it was removed until someone detected some "missing side effect" because it didn't run? Doesn't look like fluid-build enforces that all the dependencies for a given task _must_ exist (e.g. `compile -> commonjs -> tsc -> build:genver`; this package doesn't have `build:genver` and I don't expect that would be an issue when building).

Reply:
Yeah.  I share your concern as well.   The only option I can think about is to have each package explicit declare these "tasks", but that seems tedious and promote divergence.    Also, there is still the flip side of adding a script like "build:blah" and forgot to add it in the build rule.

I went with the "trust" the user approach to assume they would validate their change/PR to notice the problem.

However, other suggestions welcome.